pkg/verifier/factory/factory.go

```diff
@@ -52,14 +52,14 @@ func Register(name string, factory VerifierFactory) {
 // returns a single verifier from a verifierConfig
 // namespace is only applicable in K8s environment, namespace is appended to the certstore of the truststore so it is uniquely identifiable in a cluster env
 func CreateVerifierFromConfig(verifierConfig config.VerifierConfig, configVersion string, pluginBinDir []string, namespace string) (verifier.ReferenceVerifier, error) {
-	verifierName, ok := verifierConfig[types.Name]
+	verifierSpecName, ok := verifierConfig[types.SpecName]
```

Comment:
what happens if a user provides an older config without the new `SpecName`?

Reply:
I discussed with Juncheng yesterday on it. If users provide old config like: {name: sbom}, the report will look the same as before: {name: sbom, isSuccess: true}. And if users provide new config like: {name: sbom1, type: sbom}, the report should be: {name: sbom1, type: sbom, isSucess: true}. Then the change would be backward-compatible.

Reply:
When `specName` (type) is not provided, I feel `name` could act as both `name` and `type`.